mmv1/products/databasemigrationservice/terraform.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  ConnectionProfile: !ruby/object:Overrides::Terraform::ResourceOverride
+    properties:
+      state: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      mysql.password: !ruby/object:Overrides::Terraform::PropertyOverride
+        sensitive: true
+        custom_flatten: templates/terraform/custom_flatten/database_migration_service_connection_profile_mysql_password.go.erb
```

Comment:
This field shouldn't need a custom flatten as the api is mirrored 1:1. Any context on why this is needed?

Reply:
I was under the impression we needed that custom flatten for password fields to avoid diff issues -- it appeared to be the suggested implementation from when I had searched through previous questions.